pkg/controller/common/helpers.go

```diff
@@ -130,6 +132,33 @@ func MergeMachineConfigs(configs []*mcfgv1.MachineConfig, osImageURL string) (*m
 	}, nil
 }
 
+// PointerConfig ... TODO(runcom)
+func PointerConfig(ignitionHost, role string, rootCA []byte) (ign2types.Config, error) {
```

Comment:
we might want to unify this with the very same function the installer has

Reply:
Would be nice, but doesn't seem required.  (I often follow a rule of thumb that having something in 2 places isn't a serious problem, but if it ever needs to become 3 that's when it's really required to de-duplicate)

That said let's at least add a comment in both places like:
```
// NOTE: If you change this, you also need to change the pointer configuration in openshift/installer, see
// <installer git link>
```